modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItUuidTest.java

```diff
@@ -239,7 +238,7 @@ public void testUuidTypeCoercion() {
 
     @Test
     public void testTypeCoercionInDml() {
-        sql("INSERT INTO t VALUES (1, ?)",  UUID_1.toString());
+        sql("INSERT INTO t (id, uuid_key) VALUES (1, ?)",  UUID_1.toString());
```

Comment:
the following case doesn't work:
`INSERT INTO t (id, uuid_key) VALUES (10, null), (20, '" + UUID.randomUUID() + "')`

Reply:
@ygerzhedovich 
It should not work because types in VALUES are do not match:
`
sql("CREATE TABLE integers(i INTEGER primary key, v2 INTEGER)");

sql("INSERT INTO integers VALUES (1, null), (4, '4')"); //1
INSERT INTO integers VALUES (2, 2), (3, '3') // 2
`
Both 1 and 2 Trigger Values passed to VALUES operator must have compatible types.

But in case of a custom data type there is a bug that prevents this error from triggering. I already have a fix and file an issue later.